sera.go

```diff
@@ -41,9 +42,15 @@ const (
 
 // Represents the json config for sera
 type Config struct {
-	Server  string `json:"server"`
-	Syslog  bool   `json:"syslog"`
-	Verbose bool   `json:"verbose"`
+	Server             string `json:"server"`
+	Syslog             bool   `json:"syslog"`
+	Verbose            bool   `json:"verbose"`
+	IgnoreUnacquirable bool   `json:"-"`
```

Comment:
you can just make this unexported and it wouldn't be picked up the JSON marshaller